### DISCUSSION TITLE
Docs on vendor integration?

### DISCUSSION THREAD

**Discussion by vojtechkral** [satimoto/satimoto#78] [General]
Hi,
Satimoto has worked pretty smoothly for me so far. That being said, given that charging vendor support is still fairly spotty, I was wondering if there are any docs about how the integeration works, how is conversion/clearing to fiat for the vendor done, and what does it generally take to integrate with a new vendor.
I wasn't able to find anything on the website or blog so far...
Thanks!

**Comment by dangeross** [comment]
I'll try to write this up in a more transparent way and put it on the website. Are you asking for a general overview or more technical? Protocols, etc

**Comment by vojtechkral** [comment]
idk, I suppose general overview is fine for start? :) whatever you feel up to...
